src/griffe_warnings_deprecated/extension.py

```diff
@@ -16,7 +16,7 @@
 def _deprecated(obj: Class | Function) -> str | None:
     for decorator in obj.decorators:
         if decorator.callable_path in _decorators:
-            return str(decorator.value).split("(", 1)[1].rstrip(")").rsplit(",", 1)[0].lstrip("f")[1:-1]
+            return str(decorator.value.arguments[0]).lstrip("f")[1:-1]
```

Comment:
Nice! We can even remove the `.lstrip` call since it was meant to remove any `f` prefix and the wrapping quotes, which isn't necessary anymore. Instead the wrapping `str` call should be replaced with `ast.literal_eval`.

Reply:
Sorry, just saw this now. Yeah good call on removing the test with an f-string since we don't want to support it.